README.md

```diff
@@ -34,11 +38,19 @@ import {
 } from 'jotai-devtools';
 ```
 
-### Migration guide from `jotai/devtools` to `jotai-devtools`
+### Migration guide from `jotai/react/devtools` to `jotai-devtools`
```

Comment:
Note that, once this is (pre)released, I will drop `jotai/react/devtools`.

Reply:
That makes sense! I'll update the docs once Jotai `v2` is released then.